docs/tutorials/telescope_prog_intro.md

```diff
@@ -0,0 +1,254 @@
+# Introduction
+
+This tutorial assumes you have a basic understanding of how Airflow works, and the process of creating Airflow DAGs.  [Airflow provide a tutorial on the introductory concepts.](https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html)  Many workflows in the Academic Observatory follow similar design patterns like extract, transform, load (ETL). Telescopes aim to reduce some of the boiler plate code required in order to write some of the workflows used in the Academic Observatory platform.
+
+## Telescopes
+
+Each workflow in the Academic Observatory consists of set of tasks.  The tasks are organised as nodes on a directed acyclic graph (DAG) which describe the order of execution, and dependencies. Telescope tasks are Python functions with a specific signature, that are turned into Airflow PythonOperator objects.  The exception is the sensor tasks, which are Airflow Sensor objects, e.g., ExternalTaskSensor.
+
+The telescope class groups tasks into:
+- sensor tasks,
```

Comment:
I'm not sure this is displaying correctly, did you mean for this to be a bullet point list?